source/dubregistry/mongodb.d

```diff
@@ -0,0 +1,20 @@
+module dubregistry.mongodb;
+
+import vibe.db.mongo.client : MongoClient;
+import vibe.db.mongo.mongo : connectMongoDB;
+import vibe.db.mongo.settings : MongoClientSettings, MongoAuthMechanism, parseMongoDBUrl;
+
+MongoClientSettings mongoSettings;
+
+shared static this()
+{
+	import std.process : environment;
+	string mongodbURI = environment.get("MONGODB_URI", "mongodb://127.0.0.1");
+	parseMongoDBUrl(mongoSettings, mongodbURI);
+	mongoSettings.authMechanism = MongoAuthMechanism.scramSHA1;
```

Comment:
Default for Mongo3 (released since two years) - many cloud instances don't support `none` anymore:

https://github.com/rejectedsoftware/vibe.d/issues/1785